music21/corpus/__init__.py

```diff
@@ -246,12 +246,11 @@ def getComposer(composerName, fileExtensions=None):
     )
 
 
-@property
```

Comment:
can you explain this change?  Thanks!

Reply:
Coveralls showed that the doctest wasn't running, and since all the doctest did was attempt to call the property, it suggested to me it wasn't behaving as intended, as here:
```
>>> corpus.noCorpus
<property object at 0x11e4dd450>
```
Is that the intended use?